### PR TITLE
CM: Downgrade spammy log messages for cluster add and remove operations 

### PR DESCRIPTION
**Pull request by yanavlasov**
Commit Message:
Downgrade spammy log messages for cluster add and remove operations to debug level.

Additional Description: for large config updates it causes a flood of messages into the log file. It is both noisy and shows up in CPU profiles.

Risk Level: Low
Testing: Unit Tests
Docs Changes: N/A
Release Notes: N/A
Platform Specific Features: N/A

Signed-off-by: Yan Avlasov <yavlasov@google.com>

### CODE REVIEW

File: source/common/upstream/cds_api_impl.cc [hunk 1]
```diff
@@ -96,10 +100,13 @@ void CdsApiImpl::onConfigUpdate(const std::vector<Config::DecodedResourceRef>& a
   for (const auto& resource_name : removed_resources) {
     if (cm_.removeCluster(resource_name)) {
       any_applied = true;
-      ENVOY_LOG(info, "cds: remove cluster '{}'", resource_name);
+      ENVOY_LOG(debug, "cds: remove cluster '{}'", resource_name);
     }
   }
 
+  ENVOY_LOG(debug, "cds: added/updated {} cluster(s), {} skipped unmodified cluster(s)",
```
Comment (mattklein123) [inline source/common/upstream/cds_api_impl.cc:107]: Maybe make this one info? I don't think that should be a perf issue?

Also I would suggest but up to you:

```suggestion
  ENVOY_LOG(debug, "cds: added/updated {} cluster(s), skipped {} unmodified cluster(s)",
```

**Review by mattklein123 — CHANGES_REQUESTED** [review]
Thanks LGTM with small comment.

/wait

File: source/common/upstream/cds_api_impl.cc (hunk 1 above)
Comment (yanavlasov) [reply source/common/upstream/cds_api_impl.cc]: done

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Pull request merged**